standalone/static/app.js

```diff
@@ -62,22 +62,34 @@ const App = () => {
   });
 
   const handleRepoUrlChange = (event) => {
-    console.log("handleRepoUrlChange", event.target.value);
-    setRepoUrl(event.target.value);
-    const repoName = utils.getRepoNameFromUrl(event.target.value);
+    const value = event.target.value;
+    console.log("handleRepoUrlChange", value);
+    const repoName = utils.getRepoNameFromUrl(value);
     if (repoName) {
       console.log("repoName", repoName);
       setCurrentRepoName(repoName);
     }
+    if (value && !isValidGithubUrl(value)) {
+      setRepoUrlError("Please enter a valid GitHub repository URL.");
+    } else {
+      setRepoUrlError("");
+    }
   };
   const handleUsernameChange = (event) => {
     setUsername(event.target.value);
   };
   const handlePatChange = (event) => {
     setPat(event.target.value);
   };
+  function isValidGithubUrl(url) {
```

Comment:
this is too strict. what if the user copies from the url bar like `https://github.com/stakwork/stakgraph` ... it doesnt end with .git?